content/_data/changelogs/lts.yml

```
@@ -9733,6 +9733,506 @@
     message: |-
       Restore context menus of model links in build history views and administrative monitors (regression in 2.402).
 
+- version: "2.426.1"
+  date: 2023-11-15
+  banner: >
+      Short tags (without "jdk" in them) such as <code>jenkins/jenkins:2.414.3-alpine</code> are using Java 17 and not Java 11 like previously.
+      If you need to keep using Java 11, use tags like <code>jenkins/jenkins:2.414.3-jdk11</code>.
+      Also note that two new tags (2.414.3-alpine-jdk17 & 2.414.3-slim-jdk17) have been published without any content change a week later than the original ones.
+  lts_predecessor: "2.414.3"
+  lts_baseline: "2.426"
+  changes: # compared to lts_baseline 2.426 - extracted from the RC commit(s)
+
+  - type: major bug
+    category: bug
+    pull: 8422
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-71252][JENKINS-70793] Multiple form validation fixes"
+    references:
+      - issue: 71252
+      - issue: 70793
+    message: |-
+      Show form validation results for form elements that are initially hidden.
+      Remove previous form validation errors when the form validation is updated with new content (regression in 2.355).
+  - type: major bug
+    category: bug
+    issue: 72170
+    pull: 8602
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-72170] fix nested hetero-list entries with mixture of inputs and buttons"
+    message: |-
+      Fix multibranch Pipeline <strong>Add source</strong> and other uses that mix inputs and buttons (regression in 2.422).
+  - type: major bug
+    category: bug
+    issue: 72163
+    authors:
+      - basil
+    pr_title: "[JENKINS-72163] Retry on initial connection failure occurs in one entrypoint but not the other"
+    references:
+      - url: https://github.com/jenkinsci/remoting/pull/675
+        title: Remoting PR 675
+    message: |-
+      Add <code>sleep</code> call when <code>-noReconnect</code> is not specified for Kubernetes agents.
+  - type: major bug
+    category: bug
+    issue: 65368
+    authors:
+      - sunweisheng
+    pr_title: "Remoting agent.jar does not work behind proxy"
+    message: |-
+      Add proxy support for Remoting.
+  - type: major bug
+    category: regression
+    pull: 8564
+    issue: 71937
+    authors:
+      - Vlatombe
+    pr_title: "[JENKINS-71937] Fix deprecated 'Slave' constructor"
+    message: |-
+      Fix agent allocation due to label issue detected by vSphere Cloud plugin (regression in 2.421).
+  - type: major bug
+    category: bug
+    pull: 8613
+    issue: 72189
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-72189] fix drag&drop handle for existing repeatables"
+    message: |-
+      Fix drag and drop handle for existing repeatables.
```

State the version where the regression was introduced.
```suggestion
      Fix drag and drop handle for existing repeatables (regression in 2.335).
```